### ISSUE TITLE
Set focus to scroll target after anchor click

### ISSUE THREAD

**Issue by florianbrinkmann** [pixolin/scroll-to-anchor#6]
Hi Bego,

just wanted to leave a note on an accessibility aspect: the keyboard focus should be set to the scroll target after an anchor link was clicked. At the moment, a keyboard user would jump back to the next focus point after the anchor instead of the next focus point after the scroll target. There is a CSS tricks article about that with possible solutions: https://css-tricks.com/smooth-scrolling-accessibility/

Cheers,
Florian

**Comment by pixolin** [comment]
Hi Florian,

my reply comes a little late as I was busy with other stuff, but I want to thank you for your note on accessibility! I've added a [few lines of code](https://github.com/pixolin/scroll-to-anchor/blob/master/js/scroll-to-anchor.js#L52) yesterday to set the focus after the anchor und updated the plugin in the WordPress plugin directory.

Best regards, Bego
